backend/app/model/user_model.py

```diff
@@ -0,0 +1,32 @@
+from pydantic import BaseModel, EmailStr, Field
+from typing import Optional, List
+from datetime import datetime
+
+class User(BaseModel):
+    id: str = Field(..., description="Unique identifier for the user")
+    email: EmailStr = Field(..., description="User's email address")
+    username: str = Field(..., min_length=3, max_length=50, description="User's chosen username")
+    password_hash: str = Field(..., description="Hashed password")
+    full_name: Optional[str] = Field(None, description="User's full name")
+    is_active: bool = Field(default=True, description="Whether the user account is active")
+    created_at: datetime = Field(default_factory=datetime.utcnow, description="Timestamp of account creation")
+    last_login: Optional[datetime] = Field(None, description="Timestamp of last login")
+    watch_history: List[str] = Field(default_factory=list, description="List of IDs of watched content")
```

Comment:
curious to hear what made you decide to add the watch_history and bookmark lists to the user class/table